Assets/Scripts/Game/UI.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UI : MonoBehaviour
+{
+    [SerializeField] private Text _textScore;
+
+    private void Start()
+    {
+        GetComponent<Wallet>().OnGetingScore += PrintScore;
```

Comment:
https://trello.com/c/QAQLkgbR/7-%D0%B3%D0%B4%D0%B5-%D0%BB%D1%83%D1%87%D1%88%D0%B5-%D0%BF%D0%BE%D0%B4%D0%BF%D0%B8%D1%81%D1%8B%D0%B2%D0%B0%D1%82%D1%8C%D1%81%D1%8F